src/applications/benefit-eligibility-questionnaire/containers/App.jsx

```diff
@@ -6,12 +6,16 @@ import RoutedSavableApp from 'platform/forms/save-in-progress/RoutedSavableApp';
 import SubmitHelper from '../components/SubmitHelper';
 import formConfig from '../config/form';
 
-export default function App({ location, children }) {
-  const breadcrumb = location.pathname.split('/')[1];
+const capitalizeFirstLetter = string => {
+  return string.charAt(0).toUpperCase() + string.slice(1);
+};
 
-  const capitalizeFirstLetter = string => {
-    return string.charAt(0).toUpperCase() + string.slice(1);
-  };
+export default function App({ location, children }) {
+  const path = location.pathname.split('/')[1];
+  const breadcrumb =
+    path === 'confirmation'
+      ? 'Benefit Questionnaire Results'
```

Comment:
I think the text may have been updated again on the Figma. I see "Your benefits and resources".